rdrf/rdrf/db/contexts_api.py

```diff
@@ -123,3 +123,10 @@ def get_context(self, context_id, patient_model):
             return rdrf_context_model
         except RDRFContext.DoesNotExist:
             raise RDRFContextError("Context does not exist")
+
+    def get_previous_contexts(self, context_id, patient_model):
+        content_type = ContentType.objects.get_for_model(patient_model)
+        return RDRFContext.objects.filter(
```

Comment:
I think it would better if this filtered by `ContextFormGroup` as well, as I see no reason for getting previous contexts of other Context Form groups. Indeed, the code is manually filtering them out after calling this method.

Reply:
Updated.